### PR TITLE
dev-python/notebook + dependencies: New package + version bumps

### PR DESCRIPTION
**Pull request by marbre**
@jlec  @SoapGentoo 

notebook-4.2.0 requires nbconvert-4.2.0. Therefore I wrote an ebuild for entrypoints which is a new dependency of nbconvert. Furthermore, I bumped several old versions to EAPI=6.

### CODE REVIEW

**Comment by SoapGentoo** [comment]
@marbre looks good to me. From what I gather from the python crowd, you don't add py3.3 support for new ebuilds anymore.
@jlec your opinions?

**Comment by marbre** [comment]
@SoapGentoo I agree. Removed python3_3 from the new versions/ebuilds.

**Comment by idella** [comment]
ph3.3 is supposed to be retained in ebuilds that already have it.
You do not add py3.3 support to new packages. py3.3 is old and passee but still supported by gentoo. It is only deprecated, not banned.

File: dev-python/nbconvert/nbconvert-4.1.0.ebuild [hunk 1]
```diff
@@ -1,8 +1,8 @@
-# Copyright 1999-2015 Gentoo Foundation
+# Copyright 1999-2016 Gentoo Foundation
 # Distributed under the terms of the GNU General Public License v2
 # $Id$
 
-EAPI=5
+EAPI=6
```
Comment (jlec) [inline dev-python/nbconvert/nbconvert-4.1.0.ebuild]: @marbre don't do implace EAPI bumps in the tree. It's okay in our overlay, but it's against tree policies.

File: dev-python/notebook/notebook-4.0.6.ebuild [hunk 2]
```diff
@@ -1,8 +1,8 @@
-# Copyright 1999-2015 Gentoo Foundation
+# Copyright 1999-2016 Gentoo Foundation
 # Distributed under the terms of the GNU General Public License v2
 # $Id$
 
-EAPI=5
+EAPI=6
```
Comment (jlec) [inline dev-python/notebook/notebook-4.0.6.ebuild]: Here as well

File: dev-python/notebook/notebook-4.1.0.ebuild [hunk 3]
```diff
@@ -2,7 +2,7 @@
 # Distributed under the terms of the GNU General Public License v2
 # $Id$
 
-EAPI=5
+EAPI=6
```
Comment (jlec) [inline dev-python/notebook/notebook-4.1.0.ebuild]: Here as well

**Comment by jlec** [comment]
Beside the EAPI thing it is good to go.

**Comment by marbre** [comment]
@jlec I will hopefully remember this next time. Removed the EAPI bumps.

**Pull request merged**